saleor/attribute/__init__.py

```diff
@@ -27,6 +27,14 @@ class AttributeInputType:
         MULTISELECT,
     ]
 
+    # list of the input types that are unique per instances
+    TYPES_WITH_UNIQUE_VALUES = [
```

Comment:
Types with unique values are those that are not in `TYPES_WITH_CHOICES` I guess? We could use it and inverse the if-condition.

Reply:
We agreed to leave it as it is.